helper/schema/resource_importer_test.go

```diff
@@ -14,3 +18,168 @@ func TestInternalValidate(t *testing.T) {
 		t.Fatal("ResourceImporter should not allow State and StateContext to be set")
 	}
 }
+
+func TestImportStatePassthroughWithIdentity(t *testing.T) {
+	// shared among all tests, defined once to keep them shorter
+	identitySchema := map[string]*Schema{
+		"email": {
+			Type:              TypeString,
+			RequiredForImport: true,
+		},
+		"region": {
+			Type:              TypeString,
+			OptionalForImport: true,
+		},
+	}
+
+	tests := []struct {
+		name                 string
+		idAttributePath      string
+		resourceData         *ResourceData
+		expectedResourceData *ResourceData
+		expectedError        string
+	}{
```

Comment:
We should also write a failing test case where the identity attribute path does exist, but it's not a string 🙂